app/javascript/packs/order_screen.js

```diff
@@ -252,7 +252,7 @@ document.addEventListener('turbolinks:load', () => {
               throw new Error(error.body.text);
             } catch(e) {
               /* eslint-disable no-undef */
-              Raven.captureException(e);
+              Sentry.captureException(e);
               /* eslint-enable */
```

Comment:
```suggestion
              /* eslint-enable no-undef */
```

I know you didn't touch this but imo it should be like this